device/samples/device_methods.js

```diff
@@ -10,15 +10,9 @@ var Client = require('azure-iot-device').Client;
 var client = null;
 
 function main() {
-    // receive the IoT Hub connection string as a command line parameter
-    if(process.argv.length < 3) {
-        console.error('Usage: node device_methods.js <<IoT Hub Device Connection String>>');
-        process.exit(1);
-    }
-
     // open a connection to the device
-    var deviceConnectionString = process.argv[2];
-    client = Client.fromConnectionString(deviceConnectionString, Protocol);
+    var connectionString = process.env.IOTHUB_CONNECTION_STRING;
```

Comment:
>process.env.IOTHUB_CONNECTION_STRING [](start = 27, length = 36)

Typically this has referred to the connection string used for the hub not for a device.